### PR TITLE
Add cli for submit tx transaction

### PR DESCRIPTION
**Pull request by Anmol1696**
## Description

Add cli command for `SubmitTx` message server method.

closes: #2062

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Comment by Anmol1696** [comment]
Tests seem to failing in pulling the image for the docker for v4. Would it make sense to either just build the docker image in a step above the upgrade test?
Or handle the images better. Ideally since this is v4, we can just build it in the same action, since it is not going to be a long lasting CI anyways.

File: modules/apps/27-interchain-accounts/controller/client/cli/tx.go [hunk 1]
```diff
@@ -68,3 +76,65 @@ the associated capability.`),
...
 	return cmd
 }
+
+func newSubmitTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "submit [connection-id] [path/to/packet_msg.json]",
+		Short: "Submit an interchain account txn on the provided connection.",
+		Long: strings.TrimSpace(`Submits pre-built packet data containing messages to be executed on the host chain 
+from an authentication module and attempts to send the packet. Packet data is provided as json, file or string. An 
+appropriate absolute timeoutTimestamp must be provided with flag {packet-timeout-timestamp}, along with a timeoutHeight
```
Comment (colin-axner) [inline modules/apps/27-interchain-accounts/controller/client/cli/tx.go:86]:
```suggestion
appropriate relative timeoutTimestamp must be provided with flag {packet-timeout-timestamp}, along with a timeoutHeight
```

File: modules/apps/27-interchain-accounts/controller/client/cli/tx.go [hunk 2]
```diff
@@ -68,3 +76,65 @@ the associated capability.`),
 
 	return cmd
 }
+
+func newSubmitTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "submit [connection-id] [path/to/packet_msg.json]",
+		Short: "Submit an interchain account txn on the provided connection.",
+		Long: strings.TrimSpace(`Submits pre-built packet data containing messages to be executed on the host chain 
+from an authentication module and attempts to send the packet. Packet data is provided as json, file or string. An
```
Comment (colin-axner) [inline modules/apps/27-interchain-accounts/controller/client/cli/tx.go:85]:
```suggestion
and attempts to send the packet. Packet data is provided as json, file or string. An 
```

**Review by colin-axner — APPROVED** [review]
Great work @Anmol1696. Thanks for the contributions! :)

**Review by chatton — APPROVED** [review]
LGTM, upgrade test should no longer run when changes from main are pulled in 👍

**Review by damiannolan — APPROVED** [review]

**Pull request merged**